catroid/src/main/java/org/catrobat/catroid/io/StorageHandler.java

```diff
@@ -613,9 +613,8 @@ public static void copyDirectory(File destinationFile, File sourceFile) throws I
 	}
 
 	private void fixFolderStructureForSupportProject(String projectName, String sceneName) throws IOException {
-		projectName = UtilFile.encodeSpecialCharsForFileSystem(projectName);
-		sceneName = UtilFile.encodeSpecialCharsForFileSystem(sceneName);
-		String projectPath = buildProjectPath(projectName);
+		String projectNameEscaped = UtilFile.encodeSpecialCharsForFileSystem(projectName);
+		String projectPath = buildPath(Constants.DEFAULT_ROOT, projectNameEscaped);
```

Comment:
There is a helper method to create an escaped path for a project.
[`Utils.buildProjectPath(String projectName)`](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/utils/Utils.java#L349-L351)

Thanks for the awesome commit message! It was easy peasy lemon squeezy to review! :bowing_man: